addons/account/static/src/views/res_company_onboarding_form_controller.js

```diff
@@ -9,4 +9,10 @@ export default class CompanyOnboardingFormController extends OnboardingStepFormC
     get stepName() {
         return "account.onboarding_onboarding_step_company_data";
     }
+    get stepIsValidationConditionMet() {
```

Comment:
Could we instead make it a regular method to have it async instead of a property?
Something like "async isStepValid" ?

Would prevent this strange syntax inside the method.

Reply:
Well, the others are getters, and why I asked for the opposite change, sorry @duong77476-viindoo , @awa-odoo wins.

Reply:
Yes, actually this is my first approach